Dockerfile-dev

```diff
@@ -12,7 +12,7 @@ RUN adduser --home /code --shell /bin/bash --disabled-password --gecos "" decidi
   && adduser decidim sudo \
   && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
-RUN echo 'Defaults secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/gems/bin"' > /etc/sudoers.d/secure_path
+RUN echo 'Defaults secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bundle/bin"' > /etc/sudoers.d/secure_path
 RUN chmod 0440 /etc/sudoers.d/secure_path
 
 RUN mkdir /gems && chown decidim:decidim /gems
```

Comment:
Just noticed that I forgot to update this back, I wonder how the image is working for me then :O.

Reply:
@deivid-rodriguez The intent was try to keep gems that comes preinstalled with the image apart from gems added later by the user. 

Gems folder is mounted inside a volume in order to allow the user to add gems without the need of building the image each time the gem file changes.

If you have tested it and you're able to install gems as a user and they're available for all instances then we can accept it.

Additionally, if gems folder is removed we should also change all docker-compose.yml files and adjust the gems volume to the new location.

Reply:
> @deivid-rodriguez The intent was try to keep gems that comes preinstalled with the image apart from gems added later by the user.

I see the intent now, not sure about why though.

> Gems folder is mounted inside a volume in order to allow the user to add gems without the need of building the image each time the gem file changes.

Yup! :+1: 

> If you have tested it and you're able to install gems as a user and they're available for all instances then we can accept it.

Can you confirm that it works for you too?

> Additionally, if gems folder is removed we should also change all docker-compose.yml files and adjust the gems volume to the new location.

Yup! :+1: